### PR TITLE
Feat/home 64 new node red schedules

### PR DESCRIPTION
**Pull request by mgumis15**

### CODE REVIEW

File: views/BuzzerView/components/TurnOnBuzzerSelect/TurnOnBuzzerSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    }
+    onToggle();
+  };
+
+  return (
+    <>
+      <CheckBox
+        checked={isBuzzerTurnOnAutomatically}
+        onPress={onCheckboxPress}
+        label="Włączaj automatycznie brzęczyk, jezeli czujnik wykryje ruch"
```
Comment (mateusz-kleszcz) [inline views/BuzzerView/components/TurnOnBuzzerSelect/TurnOnBuzzerSelect.tsx:77]:
```suggestion
        label="Włączaj automatycznie alarm, jeżeli czujnik wykryje ruch"
```
alarm or brzęczyk? 🤔

File: views/SprinklerView/SprinklerView.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  ) as Sprinkler;
+
+  const sprinklerActionBaseParams = {
+    location,
+    nodeId,
+    deviceId,
+    action: 'set',
+    botHash,
+    botId,
+  };
```
Comment (mateusz-kleszcz) [inline views/SprinklerView/SprinklerView.tsx:33]: It's safer to pass `action` parameters separately in each API request. Also, you don't have to destructure values and move them to the new object.

File: views/SprinklerView/TurnOnFanSelect/TurnOnSprinklerSelect.tsx [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+  useInputValue,
+  useSelectValue,
+} from '../../../.storybook/stories';
+import { selectDevicesWithType } from '../../../redux/devicesSlice';
+import { useAppSelector } from '../../../redux/hooks';
+import { DeviceType, SprinklerSchedule } from '../../../types';
+import { sendAPIRequest } from '../../../utils';
+
+type TurnOnSprinklerSelectProps = {
+  sprinklerBaseParams: any;
```
Comment (mateusz-kleszcz) [inline views/SprinklerView/TurnOnFanSelect/TurnOnSprinklerSelect.tsx:18]: Set proper type to fanBaseParams like TempSensorBaseParams in [TempSensorView.tsx](https://github.com/inzynierka-home-iot/frontend-rncli/pull/21/files#diff-df6eb1cf97e6dff09228f26e4c6502f320c9454dd41bd61477c99a5f9e95cac8)

**Review by mateusz-kleszcz — COMMENTED** [review]
The code looks well, but I can't run it, probably because of the `yarn.lock` changes. Try to run `yarn install` before the push

File: views/SprinklerView/TurnOnFanSelect/TurnOnSprinklerSelect.tsx [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+    }
+    onToggle();
+  };
+
+  return (
+    <>
+      <CheckBox
+        checked={isSprinklerTurnOnAutomatically}
+        onPress={onCheckboxPress}
+        label="Włączaj automatycznie zraszacz, jezeli czujniki światła i wilgotności uzyskają ustawione wartości."
```
Comment (Phabes) [inline views/SprinklerView/TurnOnFanSelect/TurnOnSprinklerSelect.tsx:106]:
```suggestion
        label="Włączaj automatycznie zraszacz, jeżeli czujniki światła i wilgotności uzyskają ustawione wartości."
```

**Review by Phabes — CHANGES_REQUESTED** [review]
Add sprinkler to mock devices and modify existing buzzer (simply add  `schedule: {}` to it). While merging to main `useSelectValue` should raise conflict because current name is `useSelectIndex`)

**Review by mateusz-kleszcz — APPROVED** [review]

**Review by Phabes — APPROVED** [review]

**Pull request merged**